include/circt/Dialect/HW/HWOpInterfaces.td

```diff
@@ -478,14 +478,22 @@ def InnerSymbol : OpInterface<"InnerSymbolOpInterface"> {
           this->getOperation()->removeAttr(InnerSymbolTable::getInnerSymbolAttrName());
       }]
     >,
-    InterfaceMethod<"Returns an InnerRef to this operation's top-level inner symbol, which must be present.",
+    InterfaceMethod<"Returns an InnerRef to this operation through the inner symbol table hierarchy of which it resides.",
```

Comment:
"top-level" here wasn't re:root of the namespace, but clarifying this is for `fieldID=0`.